docs/production/export-and-import.md

```diff
@@ -204,10 +204,11 @@ To restore from a manual backup, the process is basically the reverse of the abo
 * Unpack to `/etc/zulip` the `settings.py` and `zulip-secrets.conf` files
   from your backups.
 
-* Restore your database from the backup using `wal-e`. If you ran
-  `initialize-database` anyway above, you'll want to run
+* If you ran `initialize-database` anyway above, you'll want to run
   `scripts/setup/postgres-init-db` to drop the initial database first.
```

Comment:
Hmm, I think this bit is actually obsolete -- we removed `initialize-database` as a thing that exists and one potentially needs to run at all.

Reply:
Oops, never mind, I had this mixed up with` initialize-voyager-db`.